homeassistant/components/freebox/alarm_control_panel.py

```diff
@@ -76,63 +72,40 @@ def __init__(
         self._command_state = self.get_command_id(
             node["type"]["endpoints"], "signal", "state"
         )
-        self._set_features(self._router.home_devices[self._id])
+
+        if self._command_arm_home:
+            self._attr_supported_features = (
+                AlarmControlPanelEntityFeature.ARM_AWAY
+                | AlarmControlPanelEntityFeature.ARM_HOME
+                | AlarmControlPanelEntityFeature.TRIGGER
+            )
+
+        else:
+            self._attr_supported_features = (
+                AlarmControlPanelEntityFeature.ARM_AWAY
+                | AlarmControlPanelEntityFeature.TRIGGER
+            )
```

Comment:
```suggestion
        self._attr_supported_features = (
            AlarmControlPanelEntityFeature.ARM_AWAY
            | AlarmControlPanelEntityFeature.ARM_HOME if if self._command_arm_home else 0 # or None
            | AlarmControlPanelEntityFeature.TRIGGER
        )
```
@MartinHjelmare 
Is some kind of one-liner possible in this situation ?

Tried this but not working.

Reply:
Try this:
```
 self._attr_supported_features = (
            AlarmControlPanelEntityFeature.ARM_AWAY
            | (AlarmControlPanelEntityFeature.ARM_HOME if self._command_arm_home else 0)
            | AlarmControlPanelEntityFeature.TRIGGER
        )
```

Reply:
Worked, thanks !